config.analysis

```diff
@@ -41,6 +48,9 @@ climo_yr1 = 6
 climo_yr2 = 10
 # the offset year to be added to simulation years
 yr_offset = 1849
+# start and end date for the analysis
+analysis_start_date = 0001-01-01_00:00:00
+analysis_end_date = 9999-01-01_00:00:00
```

Comment:
@milenaveneziani, I think I have a compromise solution.  If fields `climo_yr1`, `climo_yr2`, `timeseries_yr1` and `timeseries_yr2` exist, I will use them set default values of fields `climo_start_date`, `climo_end_date`, `timeseries_start_date` and `timeseries_end_date` in `run_analysis.py`.  But the longer-term goal would be to support general values for `*start_date` and `*end_date` that aren't just year boundaries.